go/okapi/native.go

```diff
@@ -90,6 +90,9 @@ func getLibraryCheckPaths() []string {
 			path.Join("/usr/local/lib", libName),
 		)
 	}
+	if runtime.GOOS == "linux" {
+		checkPaths = append(checkPaths, path.Join("/usr/local/lib", libName))	
+	}
```

Comment:
Note: I haven't had the chance to actually "test" this build. I'm guessing this will work, but 🤷 would be good to actually run through a test to confirm.